gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -118,6 +119,12 @@ def construct_argparser(self) -> argparse.ArgumentParser:
             help="Root directory. Results, plots and metrics will be stored in subdirectories,"
             " e.g. {output_root}/results",
         )
+        parser.add_argument(
+            "--run_visualizer",
+            action="store_true",
+            default=False,
```

Comment:
Hi Hayk, did you have a chance to test out this CLI with both true and false args to make sure it works as expected? I've found argparse boolean defaults often don't work as expected, so want to double check this part